packages/salesforcedx-apex-debugger/test/unit/adapter/apexDebugVariablesHandling.test.ts

```diff
@@ -358,6 +385,8 @@ describe('Debugger adapter variable handling - unit', () => {
 
       expect(container).to.be.ok;
       expect(container).to.be.instanceOf(CollectionReferenceContainer);
+      expect(container!.getNumberOfChildren()).to.equal(1);
```

Comment:
Is the `'!'` (exclamation mark) really necessary given the assert in the previous line?

Reply:
Yea, otherwise it complains container could be undefined.